lib/nbdkit.ml

```diff
@@ -26,14 +26,22 @@ open Unix_utils
 
 open Utils
 
+let nbdkit_cmd : string ref = ref ""
+
 let is_installed =
-  let test = lazy (Sys.command "nbdkit --version >/dev/null 2>&1" = 0) in
-  fun () -> Lazy.force test
+  match Config.nbdkit with
+  | Some path -> begin
+      nbdkit_cmd := path;
+      let cmd = sprintf "%s --version >/dev/null 2>&1" path in
+      let test = lazy (Sys.command cmd = 0) in
+      fun () -> Lazy.force test
+    end
+  | None -> fun () -> false
```

Comment:
I think it'd be better if you made the simpler and more obvious change:
```
  let cmd = sprintf "%s --version >/dev/null 2>&1" Config.nbdkit in
  let test = lazy (Sys.command cmd = 0) in
```

Config.nbdkit can just be a plain string, containing `"@NBDKIT@"`.

The command will still fail if `@NBDKIT@` is an empty string.